### PR TITLE
Sprite upload page

### PR DESCRIPTION
**Pull request by epeach**
Part one of moving sprite management to levelbuilder. This is just UI and doesn't connect to anything yet. Is available only at /sprites/sprite-upload, but not linked to from anywhere, so I didn't hide it behind a flag. Gif shows example usage.

example

## Follow-up work
Connecting this form to S3 and adding it to the extra-links on levelbuilder. Tracked in Jira.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/app/controllers/sprite_management_controller.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class SpriteManagementController < ApplicationController
```
Comment (ajpal) [inline dashboard/app/controllers/sprite_management_controller.rb:1]: Should we require levelbuilder permissions?

Comment (epeach) [reply dashboard/app/controllers/sprite_management_controller.rb:1]: Good call! Updated!

File: apps/src/code-studio/assets/SpriteUpload.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      <div>
+        <h1>Sprite Upload</h1>
+        <form onSubmit={this.handleSubmit}>
+          <label>
+            <h4> Sprite Category: </h4>
+            <input type="text" />
+          </label>
+          <label>
+            <h4> Select Sprite to Add to Library: </h4>
+            <input type="file" ref="uploader" onChange={this.handleChange} />
```
Comment (ajpal) [inline apps/src/code-studio/assets/SpriteUpload.js:30]: Is there a way to ensure the selected file is a png?

**Review by ajpal — APPROVED** [review]

File: apps/src/code-studio/assets/SpriteUpload.js (hunk 2 above)
Comment (epeach) [reply apps/src/code-studio/assets/SpriteUpload.js:30]: Yes! I'll add that in during the next step (uploading to S3)

Comment (epeach) [reply apps/src/code-studio/assets/SpriteUpload.js:30]: (Added it to the next PR in this task)

Comment (breville) [reply apps/src/code-studio/assets/SpriteUpload.js:30]: (Presumably using the `accept` [attribute](https://stackoverflow.com/questions/3828554/how-to-allow-input-type-file-to-accept-only-image-files)?)

Comment (epeach) [reply apps/src/code-studio/assets/SpriteUpload.js:30]: (Yup! `input type="file"` is magic)

**Pull request merged**